lib/index.js

```diff
@@ -223,6 +223,9 @@ class Hatch {
             for (let outgoing_asset_id of asset.get_dependent_assets())
                 outgoing.add(outgoing_asset_id);
         });
+        this._videos.forEach((video) => {
```

Comment:
yeah this bit's a tad confusing. Videos are dependent assets, and will be present in the `outgoing` set -- however, prior to this commit, they weren't considered part of the hatch's `asset_ids`, so this check failed